DOC-1620-Server-Size-limits-for-attributes-4.0 (#680)
tigergraph/server-docs: modules/reference/pages/configuration-parameters.adoc

* The default value is 0. When the value is 0, the threshold is dynamically decided by TigerGraph, based on current memory usage statistics.
| `TransactionSizeLimit=1073741824`

|| `FieldSizeLimit`, GPE.BasicConfig.Env
a|
* Specifies the size threshold for a single FIELD in a data record 
* The default value is 100MB. 

> | `FieldSizeLimit=5`

Author: ```suggestion
| `FieldSizeLimit=20000`
```

> * Specifies the size threshold for a single FIELD in a data record 
> * The value must be uint64_t type and in unit: Byte.The default value is 104,857,600 bytes i.e. 100MB and maximum value is 18446744073709551615 bytes i.e. 17,179,869,184 MB. 

Reviewer: ```suggestion
* Specifies the maximum number of bytes (as an unsigned integer `UINT`) for a single field in a data record
* The default is 104,857,600 = 100 MB. The maximum is 2^64 -1 = 16 exabytes
```
Comments:
@anjaliIthapeTG 
1. uint64_t is a C++ data type, not a GSQL data type.  Our equivalent type is UINT.
2. A big flaw in this table is that it does not systematically point out the data type and default value. The data type should be expressed concisely in the description.
3. I used Google to tell me the significance of 18446744073709551615. This is an enormous number; no one will have this much disk space. Knowing that is is 2^64 - 1 simplify means its the largest possible unsigned integer in a 64-bit field. The number "64" is a lot more informative to readers than  18446744073709551615. "Exabyte" also tells them, "wow, I'll never hit that limit."

|| `ContainerLengthLimit`, GPE.BasicConfig.Env
a|
* Specifies the threshold for the maximum number of entries allowed in a container
* The default value is 1M

> | `ContainerLengthLimit`

Author: ```suggestion
| `ContainerLengthLimit=10000`
```

> * Specifies the threshold for number of entries allowed in a container

Author: ```suggestion
* Specifies the maximum number of entries allowed in a container
```

> * The value must be uint64_t type and in unit: Byte.The default value is 1,048,576 bytes i.e. 1MB and maximum value is 18446744073709551615 bytes i.e. 17,179,869,184 MB. 

Author: ```suggestion
* The default value is 1,048,576 bytes = 1MB .The maximum is 2^64 -1 = 16 exabytes
```

> * The value must be uint64_t type and in unit: Byte.The default value is 1,048,576 bytes i.e. 1MB and maximum value is 18446744073709551615 bytes i.e. 17,179,869,184 MB. 

Reviewer: Analogous comments.
